recipes/yt/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install --single-version-externally-managed --record=record.txt
+if errorlevel 1 exit 1
```

Comment:
Just these two lines for now. We don't want to suggest to people that this works until it does. Make sense?